server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -73,6 +79,10 @@ object NettyConfig {
     shutdownEventLoopGroupOnClose = true,
     socketBacklog = 128,
     socketKeepAlive = true,
+    requestTimeout = FiniteDuration(60, SECONDS),
+    connectionTimeout = FiniteDuration(60, SECONDS),
+    socketTimeout = FiniteDuration(60, SECONDS),
+    lingerTimeout = FiniteDuration(1, SECONDS),
```

Comment:
**suggestion:** Are these values taken from Netty and repeated here? Maybe we shouldn't repeat them and risking diverging from Netty, but instead use `Option` and set them only if user provides a value?

Reply:
Yes, I was thinking about something like this. Generally, Netty doesn't provide any default values, and the documentation is also very obscure. For example, I can't find any information about the default lingerTimeout, so one second is my subjective point of view.

Reply:
I think the default is "not set". A thing to check is, should this be set to a default value in case of an HTTP server and if so, to what

Reply:
I think that i don't understand. Could you repeat what exactly should i check?

Reply:
what are reasonable defaults for these timeouts in case of http servers - maybe you can take a look at the configs of other servers, such as spring, tomcat, akka http, http4s etc.

it's also possible that `linger` should be optional, as by default it's not set by netty - but maybe in case of http it should have a default value

Reply:
maybe, in fact, all of these timeouts should be optional, and then we'd set only those that are defined? wdyt, @kciesielski ?

Reply:
@adamw Yes, that was the idea I was suggesting - to keep defaults as defined internally, and only set these which are customized by the user. This would mean having them all as Option.